lua/acf/core/utilities/util_sv.lua

```diff
@@ -399,97 +399,150 @@ do -- Entity linking
 		return Result
 	end
 
-	--[[
-		Example structure of ClassLink:
+	--- @class LinkFunction
+	--- @field Ent1 table The first entity in the link
+	--- @field Ent2 table The second entity in the link
+	--- @field FromChip boolean If the link is from a chip
+	--- @return boolean? Success Whether the link was successful
+	--- @return string? Message A message about the link status
+
+	--- Represents information about storing the link on an entity
+	--- @class LinkStore
+	--- @field StoreName string Any entity of this class will store references to the other class in this attribute
+	--- @field Ordered boolean If true, maintains the link as an array rather than a lookup table
+
+	--- @class LinkOptions
+	--- @field Link LinkFunction The function to handle linking
+	--- @field Unlink LinkFunction The function to handle unlinking
+	--- @field Check LinkFunction The function to check the link status
+	--- @field ChipDelay number The delay associated with the link if done via chip
+	--- @field LinkStore1 LinkStore Information about this class stores links to the other class
+	--- @field LinkStore2 LinkStore Information about how the other class stores links to this class
+
+	local ClassLink = { }
+
+	function ACF.GetLinkDataSafe(Class1, Class2)
```

Comment:
In terms of naming convention, returning a "Safe" value often means a copy of it. In this case, the actual real value is being returned.